v1/api/swagger/components/schemas/Limits.yaml

```
@@ -17,3 +22,10 @@ properties:
     type: integer
     description: Maximum number of private networks to allow
     minimum: 0
+  InstanceTypes:
+    type: object
+    additionalProperties:
+      type: integer
+    description: Map of instance type ID to maximum number of instance types to allow
```

That's actually confusing.  "instance types" has a meaning.  This is more like instance type ID to maximum number of hosts to allow.

```suggestion
    description: Map of instance type ID to maximum number of hosts that can be created with that instance type
```